tox.ini

```diff
@@ -14,6 +14,7 @@ basedeps = mock>=1.3.0
 deps = {[testenv]basedeps}
        django
        keyring
+       jsonpickle
```

Comment:
I do think making the contrib stuff more optional, as well as improving tox so it runs the tests on the django version/python version matrix would be good. But I think given this is a bugfix maybe hold that change for a separate PR?